HACKING

```diff
@@ -0,0 +1,15 @@
+ MODIFYING OPENSSL SOURCE
+ ------------------------
+ This document describes the way to add custom modifications to OpenSSL sources.
+ 
+ If you are adding new public functions to the custom library build, you need to
+ either add a prototype in one of the existing OpenSSL header files;
+ or provide a new header file and edit Configurations/unix-Makefile.tmpl to pick up that file.
+ 
+ After that performthe following steps:
+ 
+ ./configure [your-options]
+ make update
+ make
+ 
```

Comment:
Add `make doc-nits` to make sure documentation is correct.

Reply:
before or after make?

Reply:
I personally do it after the make; but I'm not sure it matters. It should be done after `make update`, however.